docs/apps/release-notes/camino-node.md

```diff
@@ -4,11 +4,97 @@ sidebar_position: 1
 
 # Camino-Node Releases
 
-## v0.2.1-rc2 ([View on GitHub](https://github.com/chain4travel/camino-node/releases/tag/v0.2.1-rc2)){#v0_2_1-rc2}
+:::info UP-TO-DATE INFORMATION
+
+`camino-node` is under very rapid development, this page may be outdated.
+
+For more up-to-date information please check [Github Release](https://github.com/chain4travel/camino-node/releases) page.
+
+:::
+
+## v0.4.1-rc2
+
+[View on GitHub](https://github.com/chain4travel/camino-node/releases/tag/v0.4.1-rc2)
+
+- Split genesis `platformAllocation` to only stake `maxStakingAmount`
+- Add depositTx inputs for already bonded UTXOs
+
+**Full Changelog**: https://github.com/chain4travel/camino-node/compare/v0.4.1-rc1...v0.4.1-rc2
+
+---
+
+## v0.4.1-rc1
```

Comment:
This version doesn't work, shouldn't we remove it or maybe write a warning beside it ?

Reply:
The page is "release-notes", so I think should stay but we can put a warning to not use it. I will also plan to make a page where it shows the current _recommended_ node version for mainnet & testnet.

Reply:
Added a warning